runbot_travis2docker/models/runbot_repo.py

```diff
@@ -18,3 +19,27 @@ class RunbotRepo(models.Model):
         "If is empty won't push it. "
         "Don't Use this feature if you use just one runbot server.")
     travis2docker_test_disable = fields.Boolean('Test Disable?')
+    uses_weblate = fields.Boolean()
```

Comment:
You have a `weblate_url` by default from code so if you validate a `weblate_token` is good to detect a weblate build.
And we don't need activate 3 fields just 1 one is really required. (The token)
 - The weblate_url has a default.
 - The uses_weblate is not need it